backend/app/views/spree/admin/adjustment_reasons/index.html.erb

```diff
@@ -33,9 +33,7 @@
             <%= adjustment_reason.name %>
           </td>
           <td>
-            <span class="pill pill-<%= adjustment_reason.active? ? 'active' : 'inactive' %>">
-              <%= t(adjustment_reason.active? ? :active : :inactive, scope: 'spree') %>
-            </span>
+            <%= render Spree::Amin::PillComponent, status: adjustment_reason.active? ? :active : :inactive %>
```

Comment:
typo: Should be `Spree::Admin::PillComponent`

Reply:
Thanks for catching this! I guess that view is not really covered by specs 😅

Reply:
and `status` -> `state`

Reply:
Now that I see this I think we should rename `state` into `status` and `STATES` into `STATUSES` (Or even `STATUS` as the plural of Status is Status in Latin, German and even formal English, see https://english.stackexchange.com/questions/877/what-is-the-plural-form-of-status).

The problem with `state` is that it can be misunderstood as State (part of country) we even have `Spree::State` as model for a Part of Country.

Reply:
I agree, and can also be confused with the React State, since we are adopting the component terminology from that world.

Reply:
Yes, that's true :)